docs/Language/HTML/HTML.md

```diff
@@ -6,11 +6,72 @@ parent: Language
 has_children: false
 ---
 
-{{ page.title }}
-======================
+# {{ page.title }}
 
-{% include under_construction.html %}
+## What is HTML?
 
-<br>
+HTML, or Hypertext Markup Language, is the standard markup language for creating web pages and web applications. It provides the structure and layout of a webpage by using a system of tags and attributes to define elements and their relationships.
 
-<br>
\ No newline at end of file
+---
+
+1. **Markup Language**
```

Comment:
Change the following ` 1. **Markup Language** ` to ` 1. **Markup Language: ** `  Referencing a list of the following.  

- Using " : "